src/shop-app.html

```diff
@@ -18,11 +18,25 @@
 <link rel="import" href="../bower_components/iron-media-query/iron-media-query.html">
 <link rel="import" href="../bower_components/iron-pages/iron-pages.html">
 <link rel="import" href="../bower_components/iron-selector/iron-selector.html">
+<link rel="import" href="../bower_components/paper-icon-button/paper-icon-button.html">
 <link rel="import" href="shop-category-data.html">
 <!-- Since 'home' is the default route, eagerly load it. -->
 <link rel="import" href="shop-home.html">
 
+
+
 <dom-module id="shop-app">
+  <!--
+    TODO(ricitc): make these lazy-imports after
+        https://github.com/Polymer/polymer-analyzer/pull/543 lands
+        DO NOT MERGE until then :)
+  -->
+  <link rel="import" href="shop-cart-data.html">
+  <link rel="import" href="lazy-resources.html">
+  <link rel="import" href="shop-list.html">
+  <link rel="import" href="shop-detail.html">
+  <link rel="import" href="shop-cart.html">
+  <link rel="import" href="shop-checkout.html">
```

Comment:
@TimvdLippe just pointed me here, I think making these lazy imports is exactly the right solution. Then we can work towards not requiring the to be also specified as fragments in `polymer.json`.